app/controllers/export_controller.rb

```diff
@@ -185,27 +185,43 @@ def update_contentdm_credentials
 private
 
   def get_headings(collection, ids)
-    field_headings = collection.transcription_fields.order(:position).where.not(input_type: 'instruction').pluck(:label)
+
+    fields = collection.transcription_fields.order(:position).where.not(input_type: 'instruction')
     cell_headings = TableCell.where(work_id: ids).pluck('DISTINCT header')
-    @raw_headings = (field_headings + cell_headings).uniq
+
+    unqiueIdentifiers = get_unique_identifiers(fields)
+
+    @raw_headings = (fields.pluck('label') + cell_headings).uniq
     @headings = []
 
     @page_metadata_headings = collection.page_metadata_fields
     @headings += @page_metadata_headings
 
     #get headings from field-based
-    field_headings.each do |raw_heading|
-      @headings << "#{raw_heading} (text)"
-      @headings << "#{raw_heading} (subject)"
+    fields.each_with_index do |field, index|
+      @headings << "#{field.label} #{unqiueIdentifiers[index]} (text)"
+      @headings << "#{field.label} #{unqiueIdentifiers[index]} (subject)"
     end
+
     #get headings from non-field-based
-    cell_headings.each do |raw_heading|
-      @headings << "#{raw_heading} (text)"
-      @headings << "#{raw_heading} (subject)"
+    cell_headings.each_with_index do |cell_heading, index|
+      @headings << "#{cell_heading} #{unqiueIdentifiers[index]} (text)"
+      @headings << "#{cell_heading} #{unqiueIdentifiers[index]} (subject)"
     end
+
     @headings.uniq!
   end
 
+  def get_unique_identifiers(fields)
+    uniqueIdentifiers = Array.new(fields.length)
+
+    for i in 0..(fields.length - 1) do
+      uniqueIdentifiers[i] = "#{fields[i].line_number}.#{fields[i].position}"
+    end
+
+    uniqueIdentifiers
+  end
+
```

Comment:
No need to change this before we deploy, but a the Ruby `map` function can do some lovely magic here:
```ruby
unique_identifiers = fields.map { |field| "#{field.line_number}.#{field.position}"}
```